modules/nw-configure-sysctl-interface-sriov-network.adoc

```diff
@@ -0,0 +1,171 @@
+====
+
+To change the interface-level network `net.ipv4.conf.IFNAME.accept_redirects` sysctl create an additional SR-IOV network attachment with the Container Network Interface (CNI) tuning plug-in by using the following procedure.
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the {product-title} cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the `SriovNetwork` custom resource (CR) for the additional SR-IOV network attachment and insert the `metaPlugins` configuration, as in the following example CR. Save the YAML as the file `sriov-network-interface-sysctl.yaml`.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: onevalidflag <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  resourceName: policy-example-oneflag <3>
+  networkNamespace: sysctl-tuning-test <4>
+  ipam: '{ "type": "static" }' <5>
+  capabilities: '{ "mac": true, "ips": true }' <6>
+  metaPlugins : | <7>
+    {
+      "type": "tuning",
+      "capabilities":{
+        "mac":true
+      },
+      "sysctl":{ 
+         "net.ipv4.conf.IFNAME.accept_redirects": "1"
+      }
+    }
+----
+<1> A name for the object. The SR-IOV Network Operator creates a NetworkAttachmentDefinition object with same name.
+<2> The namespace where the SR-IOV Network Operator is installed.
+<3> The value for the spec.resourceName parameter from the SriovNetworkNodePolicy object that defines the SR-IOV hardware for this additional network.
+<4> The target namespace for the SriovNetwork object. Only pods in the target namespace can attach to the additional network.
+<5> A configuration object for the IPAM CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<6> Optional: The capabilities to configure for this additional network. You can specify "{ "ips": true }" to enable IP address support or "{ "mac": true }" to enable MAC address support.
+<7> Optional: The metaPlugins parameter is used to add additional capabilities to the device. In this use case set the `type` field to `tuning`. Specify the interface-level network `sysctl` you want to set in the `sysctl` field. 
+
+. Create the `SriovNetwork` resource:
++
+[source,terminal]
+----
+$ oc create -f sriov-network-interface-sysctl.yaml
+----
+
+.Verifying that the `NetworkAttachmentDefinition` CR is successfully created
+
+* Confirm that the SR-IOV Network Operator created the `NetworkAttachmentDefinition` CR by running the following command.
++
+[source,terminal]
+----
+$ oc get network-attachment-definitions -n <namespace> <1>
+----
+<1> Replace `<namespace>` with the namespace that you specified when configuring the network attachment, for example, `openshift-sriov-network-operator`.
++
+.Example output
+[source,terminal]
+----
+NAME                                  AGE
+openshift-sriov-network-operator      14m
```

Comment:
It's not recommended to use operator's names for NetworkAttachmentDefinition resources. Let's use any other name

Reply:
HI @kononovn should this not be the same namespace that you installed the SR-IOV Network Operator in. If that is the case then it would be openshift-sriov-network-operator as per https://docs.openshift.com/container-platform/4.11/networking/hardware_networks/installing-sriov-operator.html

Reply:
in the previous step you create SriovNetwork. This network should have some custom name like "mysriovnetwork". After that it would create NetworkAttachmentDefinition resource in the target namespace. This NetworkAttachmentDefinition resource will have mysriovnetwork name instread of openshift-sriov-network-operator.